source/adminguide/virtual_machines.rst

```diff
@@ -185,7 +185,7 @@ To create a VM from a template:
 
 #. Click Add Instance.
 
-#. Select a zone.
+#. Select a zone. Admin users also have to option to select a pod, cluster or host.
```

Comment:
do you mean via API (Cloudmonkey or any other tool), right?

Reply:
The list if wrt UI. See lines above line nu. 185.